samples/_bootstrap.py

```diff
@@ -0,0 +1,42 @@
+import os
```

Comment:
Let's make sure to:
1. Add the license comment to the top of the script
2. Add a small docstring explaining this script is for development purposes only and serves as a way to automate running of samples.
3. Let's rename the script to `_run_sample.py`. Mainly using the verb `run` better indicates that we are running the script. Bootstrap indicates more that we are spinning up resources which we are not.